### PR TITLE
update conv error info

### PR DESCRIPTION
**Pull request by FDInSky**
update conv error info

### CODE REVIEW

File: paddle/fluid/operators/detection/iou_similarity_op.cc [hunk 1]
```diff
@@ -24,16 +24,32 @@ class IOUSimilarityOp : public framework::OperatorWithKernel {
  protected:
   void InferShape(framework::InferShapeContext *ctx) const override {
     PADDLE_ENFORCE(ctx->HasInput("X"),
-                   "Input(X) of IOUSimilarityOp should not be null.");
+                   platform::errors::InvalidArgument(
+                       "Input(X) of IOUSimilarityOp should not be null."));
```
Comment (qingqing01) [inline paddle/fluid/operators/detection/iou_similarity_op.cc]: call `OP_INOUT_CHECK`

File: paddle/fluid/operators/detection/iou_similarity_op.cc [hunk 2]
```diff
@@ -24,16 +24,32 @@ class IOUSimilarityOp : public framework::OperatorWithKernel {
...
     auto y_dims = ctx->GetInputDim("Y");
 
-    PADDLE_ENFORCE_EQ(x_dims.size(), 2UL, "The rank of Input(X) must be 2.");
-    PADDLE_ENFORCE_EQ(x_dims[1], 4UL, "The shape of X is [N, 4]");
-    PADDLE_ENFORCE_EQ(y_dims.size(), 2UL, "The rank of Input(Y) must be 2.");
-    PADDLE_ENFORCE_EQ(y_dims[1], 4UL, "The shape of Y is [M, 4]");
+    PADDLE_ENFORCE_EQ(
+        x_dims.size(), 2UL,
+        platform::errors::InvalidArgument(
+            "The rank of Input(X) must be 2, but got dimension =%d.",
```
Comment (qingqing01) [inline paddle/fluid/operators/detection/iou_similarity_op.cc]: =前有空格，因此=后也需要空格。下同。

File: paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc [hunk 3]
```diff
@@ -129,38 +129,56 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
     auto* bias = ctx.HasInput("Bias") ? ctx.Input<Tensor>("Bias") : nullptr;
     auto* output = ctx.Output<Tensor>("Output");
 
-    PADDLE_ENFORCE_EQ(input->layout(), DataLayout::kMKLDNN,
-                      "Wrong layout set for Input tensor");
-    PADDLE_ENFORCE_NE(input->format(), MKLDNNMemoryFormat::undef,
-                      "Wrong format set for Input tensor");
+    PADDLE_ENFORCE_EQ(
+        input->layout(), DataLayout::kMKLDNN,
+        platform::errors::InvalidArgument("Wrong layout set for Input tensor"));
```
Comment (qingqing01) [inline paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc]: should set ..., but received layout is ..

Comment (FDInSky) [reply paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc]: 修改了

File: paddle/fluid/operators/detection/iou_similarity_op.cc (hunk 2 above)
Comment (FDInSky) [reply paddle/fluid/operators/detection/iou_similarity_op.cc]: 修改了

File: paddle/fluid/operators/detection/iou_similarity_op.cc (hunk 1 above)
Comment (FDInSky) [reply paddle/fluid/operators/detection/iou_similarity_op.cc]: 修改了

**Review by lijianshe02 — APPROVED** [review]

**Pull request merged**